src/config/constants.ts

```diff
@@ -561,7 +560,6 @@ export const kPandocDefaultsKeys = [
   kSectionDivs,
   "identifier-prefix",
   kTitlePrefix,
-  "strip-empty-paragraphs",
```

Comment:
Even though these are deprecated I wonder if we should keep them in here so they are forwarded (and a warning presumably generated by pandoc?).

Reply:
They were removed by Pandoc 3:

```
Remove deprecated --atx-headers option.

Remove deprecated option --strip-empty-paragraphs.
```